taskgraph/Task.py

```diff
@@ -1008,7 +945,6 @@ def __init__(
         self._func = func
         self._args = args
         self._kwargs = kwargs
-        self._cache_dir = cache_dir

```

Comment:
Huh ... interesting that this was never used and we didn't catch this before!

Reply:
I know! Must have happened when I changed from a big directory of files to the SQLite format, then just left the thing lying around. Also maybe was something hopeful from caching a precomputed file?